opentelemetry-sdk/src/metrics/instrument.rs

```diff
@@ -218,52 +217,57 @@ impl Stream {
     }
 }
 
-/// the identifying properties of a stream.
+/// The identifying properties of a instrument.
 #[derive(Debug, PartialEq, Eq, Hash)]
-pub(crate) struct StreamId {
+pub(crate) struct InstId {
```

Comment:
consider `InstrumentId` to `InstId`?  (Though maybe the spec calls for this name, I didn't look.)

Reply:
Seems reasonable 👍